### ISSUE TITLE
when file is copied with xdcp using different user, owner is always "root"

### ISSUE THREAD

**Issue by sanomiya** [xcat2/xcat-core#2637]
User _diagadmin_ has all xcat privileges. When a file is copied using this user, the ownership of the file on the target node is root instead of _diagadmin_.

```
c460mgt01:/home/diagadmin > id
uid=642490(diagadmin) gid=1043351(diagadmin) groups=1043351(diagadmin),1043350(csmadmin)
c460mgt01:/home/diagadmin > xdcp c460c034 1 /tmp/1
c460mgt01:/home/diagadmin > echo $?
0

```
```
c460c034:/tmp > ls -l 1
-rw-r--r-- 1 root root 2525 Mar 13 15:05 1
```

**Comment by gurevichmark** [comment]
I think this is the same as issue #1001

**Comment by gurevichmark** [comment]
This issue is fixed by PR #2689 
Please verify and close this issue if satisfactory.

**Comment by sanomiya** [comment]
I have no way of testing this fix in my development environment . Closing it.
Will reopen, if problem persists when this version is installed in r92.
